cmd/cli/main.go

```diff
@@ -18,15 +18,20 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
-	"net/http"
-	"os"
 
 	"github.com/spf13/cobra"
 
 	"antrea.io/resource-auditing/pkg/types"
+
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
 )
 
+// filter flags
+var getAuthor, since, until, resource, namespace, name string
```

Comment:
why is it `getAuthor`? if there is a name conflict, you probably want to change the other definition to have some naming consistency here

Reply:
There wasn't a naming conflict, since there's a tagAuthor, I thought it would be better to specify which author it referred to.

Reply:
I see.. that's fine then. Maybe you could prefix all of the flags with `get` for consistency with the tag command?